msg/TrajectorySetpoint.msg

```diff
@@ -10,6 +10,9 @@ float32[3] position # in meters
 float32[3] velocity # in meters/second
 float32[3] acceleration # in meters/second^2
 float32[3] jerk # in meters/second^3 (for logging only)
+float32[4] attitude # in quaternions
+float32[3] angular_velocity # in rad/s^2
```

Comment:
That's what [attitude setpoint](https://github.com/PX4/PX4-Autopilot/blob/253208fdd4d3b415ef9ce019d7b5475a3e103a8a/msg/VehicleAttitudeSetpoint.msg) and [rate setpoint](https://github.com/PX4/PX4-Autopilot/blob/253208fdd4d3b415ef9ce019d7b5475a3e103a8a/msg/VehicleRatesSetpoint.msg) are for. We shouldn't combine those for all use cases.

Reply:
But since the pipeline is pos_ctl sending an attitude setpoint to the att controller, how do you combine trajectories that require both? Because one of the commands gets ignored in that case... or will get jittered?